### DISCUSSION TITLE
Change Listener URL

### DISCUSSION THREAD

**Discussion by redimongo** [AzuraCast/AzuraCast#5696] [General]
So we have a ad injector software, which we want to use AzuraCast for clients who want a radio station. However instead of the listener URLs that the dashboard shows, we want to change that to the stations URL that is in our own directory system.

basically the URL would be

`https://example.com/[STATION_NAME]`

What core code do I need to change to make this possible in the backend?

**Comment by Mozitux** [comment]
I'm not sure, but maybe you could use a reverse proxy redirect to do this, it should be possible.
EDIT: It's possible by using .htacces file or even with some configuration in nginx.conf file.
